scripts/static-binaries/build-on-ubuntu.sh

```diff
@@ -14,7 +14,7 @@ set -euxo pipefail
 extra_features=${EXTRA_FEATURES:-""}
 
 protoc_version=3.15.3
-flatbuffers_version=v2.0.0
+flatbuffers_version=v22.10.26
```

Comment:
This looks like a different version than the one you mention in the README. Which one is the intended one?